docs/modules/ROOT/pages/advanced-how-tos/testing-releasing-single-component.adoc

```diff
@@ -0,0 +1,200 @@
+= Testing and Releasing a Single Component
+
+By default, Konflux treats applications as a set of dependent components. A example would frontend and backend
+components. Typically, a team would want to build these components, while testing and releasing them together
+to ensure there are compatible.
+
+However, applications can also be viewed simply as a group of non-dependent components.
+
+Consider this:
+
+* You have an application that contains 5 components.
+* You need to release a single component to fix a bug that a customer is experiencing.
+* While attempting to release, another component has a new violation therefore causing the Enterprise Contract
+policy validation to fail.
+** This violation may be due to a rule that used to be a warning but has expired and is now a violation.
+
+Wouldn't it be nice to be able to ignore that failing component and simply release the component you are targeting?
+
+Enter **Single Component Mode**!
+
+== Enabling Single Component Mode
+
+Single component mode works by analyzing the Snapshot that is being used for Integration Tests and Releasing.
+A Snapshot created due to a single component being built is the only Snapshot type that may be used for this
+mode.
+
+=== Enabling Single Component Mode for Enterprise Contract Integration Test Scenarios
+
+Single Component Mode can be enabled either by making a change to the Integration Test Scenario custom resource
+or by using the HAC UI.
+
+* Update the custom resource by adding the parameter *SINGLE_COMPONENT* with a value of *true*
+
+[source,yaml]
+.integration-test-scenario.yaml
+----
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  name: grouping-enterprise-contract-single-component
+  namespace: shebert-tenant
+spec:
+  application: grouping
+  contexts:
+    - description: Application testing
+      name: application
+  params:
+    - name: SINGLE_COMPONENT
+      value: 'true'
+  resolverRef:
+    params:
+      - name: url
+        value: 'https://github.com/konflux-ci/build-definitions'
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipelines/enterprise-contract.yaml
+    resolver: git
+----
+
+[NOTE]
+====
+You need to apply the change using *kubectl*
+====
+
+* Alternatively, you may edit the Integration Test Scenario using the HAC UI as follows:
+
+image::single-component-integration-test-scenario.png[role="border" alt="Editing your Integration Test Scenario"]
+
+=== Enabling Single Component Mode for Releases
+
+Single Component Mode can be enabled for Releases by making a change to the ReleasePlanAdmission custom resource.
+
+* Update the custom resource by adding the field *singleComponentMode* under _.spec.data_ with a value of *true*
```

Comment:
You can also add the annotation to the RPA CR page with a description and link the two pages instead.